src/plot.jl

```diff
@@ -93,47 +96,57 @@ end
     p.c.range, p.val
 end
 
-@recipe function f(c::MCMCChain.AbstractChains;
+@recipe function f(c::MCMCChain.AbstractChains, parameters::AbstractVector{Symbol}; colordim = :chain)
+    colordim != :chain && error("Symbol names are interpreted as parameter names, only compatible with `colordim = :chain`")
+    ret = indexin(parameters, Symbol.(keys(c)))
+    any(isnothing, ret) && error("Parameter not found")
+    c, Int.(ret)
+end
+
+@recipe function f(c::MCMCChain.AbstractChains,
+                   parameters::AbstractVector{<:Integer} = Int[];
                    width = 500,
                    height = 250,
                    colordim = :chain
                   )
-
     ptypes = get(plotattributes, :seriestype, (:traceplot, :mixeddensity))
-
-    # sanity check
     ptypes = ptypes isa AbstractVector || ptypes isa Tuple ? ptypes : (ptypes,)
     @assert all(map(ptype -> ptype ∈ supportedplots, ptypes))
-
-    nrows, nvars, nchains = size(c)
     ntypes = length(ptypes)
-    N = colordim == :chain ? nvars : nchains
-    layout := (N, ntypes)
-    size --> (ntypes*width, N*height)
-    indices = reshape(1:N*ntypes, ntypes, N)'
-
-    legend --> false
-
-    for (j, ptype) in enumerate(ptypes)
-        for i in 1:N
-            @series begin
-                subplot := indices[i, j]
-                colordim := colordim
-                seriestype := ptype
-                c, i
+    nrows, nvars, nchains = size(c)
+    isempty(parameters) && (parameters = colordim == :chain ? (1:nvars) : (1:nchains))
+    N = length(parameters)
+
+    if !(:corner ∈ ptypes)
+        layout := (N, ntypes)
+        size --> (ntypes*width, N*height)
+        indices = reshape(1:N*ntypes, ntypes, N)'
+        legend --> false
+        for (j, ptype) in enumerate(ptypes)
+            for (i, par) in enumerate(parameters)
+                @series begin
+                    @show parameters
```

Comment:
I believe this line still needs to go or `plot(chain)` will be a bit too verbose.

Reply:
:joy: ouch - good catch!

Reply:
fixed